specification/desktopvirtualization/resource-manager/readme.md

```diff
@@ -26,22 +26,35 @@ These are the global settings for the DesktopVirtualizationClient API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2024-03
+tag: package-2024-04
 ```
 
 
+### Tag: package-2024-04
+
+These settings apply only when `--tag=package-2024-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-04'
+input-file:
+  - Microsoft.DesktopVirtualization/stable/2024-04-03/desktopvirtualization.json
+suppressions:
+    - code: PatchIdentityProperty
+      from: desktopvirtualization.json  
+      reason: There is an error loop with the identity property. We received approval to suppress this error, but we still need to investigate error loop.
+```
```

Comment:
please suppress specific paths not the whole file. Can you also please explain what this approval is about nd who provided the approval ?

Reply:
PR where suppression was approved: https://github.com/Azure/azure-rest-api-specs/pull/26142

Reply:
Removed suppression and implemented fix